packages/i18n/src/Loader.js

```diff
@@ -221,10 +242,23 @@ EnyoLoader.prototype.loadFiles = function (paths, sync, params, callback, rootPa
 					}
 				};
 
-				this.loadManifestsSync(_root);
+				const handleAdditionalResourcesPath = (json, err) => {
+					if (found && !err && typeof json === 'object') {
+						// Overwrite the _root/path result
+						Object.assign(ret[ret.length - 1], json);
+					}
+				};
 
 				if (this.isAvailable(_root, path)) {
 					getSync(this._pathjoin(_root, path), handler);
+
+					if (this.addPaths && Array.isArray(this.addPaths)) {
+						for (let addedRoot of this.addPaths) {
```

Comment:
fyi. addPath() takes "String" or "Array". so If ILIB_ADDITIONAL_RESOURCES_PATH value is a string, this.addPaths array length 1 always here.

Reply:
I know. I wrote this for general usage.